.github/workflows/format.yml

```diff
@@ -31,6 +31,10 @@ on:
   schedule:
     - cron: '0 0 * * *'
 
+concurrency:
+  group: ${{ github.head_ref || github.run_id }}-${{ github.workflow }}
+  cancel-in-progress: true
+
```

Comment:
Nit: could you add an empty line after `concurrency:` to match the formatting in rest of this file? Thanks!